jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -109,6 +109,21 @@ private static native void checkpointJVMImpl(String imageDir,
 		});
 	}
 
+	/**
```

Comment:
I think we should also describe single thread mode:

Single thread mode is a mode in which only the the Java thread that requested a checkpoint is permitted to run. This means that global state changes are permitted without the risk of race conditions. It also implies that if one attempts to acquire a resource held by another Java thread a deadlock will occur.

Reply:
Added `SINGLE_THREAD_MODE` description.